pkgs/development/python-modules/python3-application/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, isPy3k, buildPythonPackage, fetchFromGitHub, zope_interface, twisted, ... }:
+
+buildPythonPackage rec {
+  pname = "python3-application";
+  version = "3.0.3";
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-application";
+    rev = "8209f3334c9b603fc81509333c38909755f663db";
```

Comment:
```suggestion
    rev = version
```

Reply:
In this case I want a commit (specifically [this one](https://github.com/AGProjects/python3-application/commit/695f7d769e69c84e065872ffb403157d0af282fd)) that was made after the most recent tag, which is `3.0.3` .

Reply:
Than we can't tag this as the version number. Can you just fetch that commit as a patch?